business/references/k8s_gateway_references.go

```diff
@@ -39,5 +40,16 @@ func (g K8sGatewayReferences) getConfigReferences(gw *k8s_networking_v1.Gateway)
 		}
 	}
 
+	for _, rt := range g.K8sGRPCRoutes {
+		if len(rt.Spec.ParentRefs) > 0 {
```

Comment:
Is this check necessary?  Wouldn't the range just not execute if len is 0?  I'm fairly sure that range on a nil slice is fine, as internally it would just be zero iterations.

Reply:
right